src/components/Checkbox/Checkbox.test.js

```diff
@@ -0,0 +1,56 @@
+import { render } from '@testing-library/vue';
+import { mount } from '@vue/test-utils';
+
+import Checkbox from './Checkbox.vue';
+
+describe('UI/forms/Checkbox', () => {
+  beforeAll(() => {
+    // Silence deprecation error logs from vue-test-utils. Remove this in future versions of this library:
+    console.error = jest.fn(); // eslint-disable-line no-console
+  });
+
+  it('Should be checked if given checked prop is set to true', () => {
+    const { getByTestId } = render(Checkbox, { propsData: { value: true } });
+    const inputIsChecked = getByTestId('checkbox-wrapper').className.includes('checked');
+    expect(inputIsChecked).toBeTruthy();
+  });
+
+  it('Should be unchecked if given checked prop is set to true', () => {
```

Comment:
```suggestion
  it('Should be unchecked if given checked prop is set to false', () => {
```

![](https://media.giphy.com/media/HufOeXwDOInlK/giphy.gif)

Reply:
😆  good catch! I'll fix it and the originally copy-pasta that I copied it from